src/models/wasteWater/__tests__/WasteWaterLocationTimeChart.test.tsx

```diff
@@ -0,0 +1,90 @@
+import ResizeObserver from 'resize-observer-polyfill';
+import { render, screen } from '@testing-library/react';
+import { WasteWaterLocationTimeChart } from '../WasteWaterLocationTimeChart';
+import { globalDateCache } from '../../../helpers/date-cache';
+import { WasteWaterTimeseriesSummaryDataset } from '../types';
+import { getTicks } from '../../../helpers/ticks';
+import { formatDate } from '../WasteWaterTimeChart';
+
+jest.mock('recharts', () => {
+  const OriginalModule = jest.requireActual('recharts');
+  return {
+    ...OriginalModule,
+    ResponsiveContainer: ({ children }: any) => (
+      <OriginalModule.ResponsiveContainer width={800} height={800}>
+        {children}
+      </OriginalModule.ResponsiveContainer>
+    ),
+  };
+});
+
+window.ResizeObserver = ResizeObserver;
+jest.mock('react-resize-detector');
+
+describe('WasteWaterLocationTimeChart', function () {
+  it('should display no data message when no data is provided', function () {
+    const datesOfData: string[] = [];
+    const variantNames = ['variantName1', 'variantName2'];
+
+    const dateRange = {
+      dateFrom: globalDateCache.getDay('2020-01-01'),
+      dateTo: globalDateCache.getDay('2020-01-02'),
+    };
+
+    const variants = getWasteWaterLocationTimeChartProps(datesOfData, variantNames);
+
+    render(
+      <div style={{ width: 500, height: 500 }}>
```

Comment:
Why do we need that `div` in this test, but not in the other test?

Reply:
Actually we dont. It was a leftover before the ResponsiveContainer was mocked.